lib/chains/relativePositionChain.js

```diff
@@ -0,0 +1,139 @@
+const {compose} = require('ramda');
+// Import utils
+const makeChain = require('../utils/makeChain');
+const {applyCountAndDelayAndActions, applyUntilCondition} = require('../utils/chainUtils');
+const {validate, validators} = require('../validation');
+const {getRequestType} = require('../utils/socketChainHelper');
+// Import chain composers
+const {
+	clickComposer,
+	moveToComposer,
+	repeatComposer,
+	intervalComposer,
+	makeToStringComposer,
+	makeThenComposer,
+	abandonComposer,
+	cloneComposer,
+	gettersComposer,
+	assertComposer,
+	makeToJSONComposer,
+	untilComposer,
+} = require('../composers');
+const {invalidInputMessage, relativePositionIsMalformed} = require('../texts');
+const SuitestError = require('../utils/SuitestError');
+
+/**
+ * @param {SUITEST_API} classInstance
+ */
+const relativePositionFactory = (classInstance) => {
```

Comment:
I am aware of that you are keeping existing codestyle, but I really don't know what is the purpose of it, I see no benefit:
```javascript
function relativePositionFactory(classInstance) {
const relativePositionFactory = (classInstance) => {
```